images/pod-utilities/Dockerfile

```diff
@@ -2,32 +2,34 @@ ARG POD_UTILITY=clonerefs
 FROM ppc64le/ubuntu:18.04 as builder
 
 ARG UBUNTU_VERSION=18.04
-ARG BAZEL_VERSION=3.1.0
 ARG DEBIAN_FRONTEND=noninteractive
 ARG POD_UTILITY
 ARG CO_COMMIT=master
+ARG GOLANG_VERSION=1.15.7
+ENV PATH /usr/local/go/bin:$PATH
 
-COPY nodejs.patch /tmp
+ENV GOPATH /go
+ENV PATH $GOPATH/bin:$PATH
+RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
```

Comment:
I'm wondering instead of using ubuntu whether we can use the `golang:1.15` image instead of downloading the golang tar?

Reply:
yes, even I thought so. Shall verify and resubmit.

Reply:
Verified and submitted.